jet.h

```diff
@@ -7,8 +7,58 @@ namespace correction {
 class JetCorrProvider : public CorrectionsBase<JetCorrProvider> {
 public:
      enum class UncSource : int {
-        Central = 0
+        Central = -1,
+        FlavorQCD = 0,
+        RelativeBal = 1,
+        HF = 2,
+        BBEC1 = 3,
+        EC2 = 4,
+        Absolute = 5,
+        Total = 6,
+        BBEC1_2018 = 7,
+        Absolute_2018 = 8,
+        EC2_2018 = 9,
+        HF_2018 = 10,
+        RelativeSample_2018 = 11,
+        BBEC1_2017 = 12,
+        Absolute_2017 = 13,
+        EC2_2017 = 14,
+        HF_2017 = 15,
+        RelativeSample_2017 = 16,
+        BBEC1_2016 = 17,
+        Absolute_2016 = 18,
+        EC2_2016 = 19,
+        HF_2016 = 20,
+        RelativeSample_2016 = 21
```

Comment:
- define `Total` as 0 (it is a special one that includes all other 21 sources)
- what about JER sources and their variations?

Reply:
- ok
- I though JER had to not to be splitted.

Reply:
yes, JER has one unc source. Is it considered?

Reply:
In line of principle, JER can be also splitted - there is a boolean asking if splitting or not. I thought JER was the smearing.